packages/components/src/Input/Input/Input.module.scss

```diff
@@ -0,0 +1,359 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+@import "../../../styles/form-variables";
+@import "../../../styles/forms";
+
+$input-height: 48px;
+$input-base-padding-horizontal: $spacing-sm;
```

Comment:
Could we also replace the `sm` & `md` spacing tokens with the number spacing tokens

Reply:
We could, but not the goal of these migrations.